lib/fastlane/plugin/revenuecat/actions/replace_version_number_action.rb

```diff
@@ -0,0 +1,65 @@
+require 'fastlane/action'
+require 'fastlane_core/configuration/config_item'
+require_relative '../helper/revenuecat_helper'
+
+module Fastlane
+  module Actions
+    class ReplaceVersionNumberAction < Action
+      def self.run(params)
+        previous_version_number = params[:current_version]
+        new_version_number = params[:new_version]
+        files_to_update = params[:files_to_update]
+        files_to_update_without_prerelease_modifiers = params[:files_to_update_without_prerelease_modifiers]
+        UI.user_error!("missing current version param") unless previous_version_number
```

Comment:
I'm not sure if this is needed... I would think fastlane would add some input validation using the `available_options` method but I couldn't get that to work on the tests, so I added some basic validation here for now... It would be great to get some intro into how the `available_options` work when you have time @joshdholtz

Reply:
I tested the plugin from a project and it indeed does the input validation, so I'm removing this validation here. I'm a bit concerned for not being able to test that same input from the tests though...